slackbridge/factories.py

```diff
@@ -71,24 +72,24 @@ def instantiate_bot(self, user):
 
 class UserBotFactory(BotFactory):
 
-    def __init__(self, bridge_bot_factory, slack_user, channels,
-                 target_group, nickserv_pw):
+    def __init__(self, bridge_bot_factory, slack_user, target_group,
+                 nickserv_pw):
         self.bridge_bot_factory = bridge_bot_factory
         self.slack_user = slack_user
-        self.channels = []
+        self.joined_channels = []
         self.target_group_nick = target_group
         self.nickserv_password = nickserv_pw
 
-        for channel in channels:
+        for channel_id, channel in IRCBot.channels.items():
```

Comment:
```python
for _, channel in IRCBot.channels.items()
```

Reply:
Or, better yet, ```IRCBot.channels.values()```. Doesn't seem necessary to have ```channel_id```.

Reply:
Yeah, using `.values()` would be better here, thanks.